charts/polkadot/values.yaml

```diff
@@ -4,7 +4,7 @@ image:
 
 initImage:
   repo: web3f/polkadot-deployer
-  tag: bcbefdde689113a53e657bea3973133692ca6725
+  tag: 03c71cbe8e96cb241f6d0ca7ba1103c1a82f12e5
```

Comment:
I'm wondering why we don't have sem ver tags on web3f/polkadot-deployer

Reply:
Yep, the problem is that the image is built from this repo, maybe we could use the same tagging as for the charts, wdyt?